### ISSUE TITLE
[BUG] Playwright 1.37 can not init

### ISSUE THREAD

**Issue by bba2022** [microsoft/playwright-dotnet#2663]
### System info
- Playwright Version: [v1.37]
- Operating System: [Windows 10]
- Browser: [Chromium]
- Other info:

### Source code

- [ ] I provided exact source code that allows reproducing the issue locally.

**Link to the GitHub repository with the repro**

[https://github.com/your_profile/playwright_issue_title]

or

**Test file (self-contained)**

```cs
var playwright = await Playwright.CreateAsync();


            var context = await playwright.Chromium.LaunchAsync();
            var pp = await context.NewPageAsync();
            await pp.GotoAsync("https://example.com");
**Actual**

[Describe actual behavior]
引发的异常:“System.IO.IOException”(位于 System.Private.CoreLib.dll 中)

**Comment by WilliamQue** [comment]
Meet the same thing, when  var playwright = await Playwright.CreateAsync();

**Comment by billpeace** [comment]
var exitCode = Microsoft.Playwright.Program.Main(new[] { "install" });
                if (exitCode != 0)
                {
                    throw new Exception($"Playwright exited with code {exitCode}");
                }

manual install

**Comment by mxschmitt** [comment]
Thank you for your bug report, we will issue a bugfix release for it tomorrow.

**Comment by mxschmitt** [comment]
Should be fixed in 1.37.1 which is currently getting published.
